boot/boottest/device.go

```diff
@@ -21,16 +21,30 @@ package boottest
 
 import (
 	"strings"
+
+	"github.com/snapcore/snapd/boot"
 )
 
+type mockDevice struct {
+	str  string
```

Comment:
given we are going this way I would do the next step and:

```
type mockDevice struct {
   bootSnap string
   mode string
   uc20 bool
}
```

and do the splitting early.

Reply:
This is done now.